### ISSUE TITLE
nvda is skipping empty columns when navigating with ctrl+alt+left/right arrows

### ISSUE THREAD

**Issue by beqabeqa473** [nvaccess/nvda#10615]
hello all.

in recent snapshots of nvda, i discovered some strange issue.

in multy column listviews, nvda is skipping empty cells when navigating with ctrl+alt+left/right arrows.

to better observe a situation, try to run code below and try to navigate with in first item.

```
import wx

class myForm(wx.Frame):
	def __init__(self):
		wx.Frame.__init__(self, None, wx.ID_ANY, "List Control Tutorial")
		panel = wx.Panel(self, wx.ID_ANY)
		self.list_ctrl = wx.ListCtrl(panel, size=(-1,100), style=wx.LC_REPORT                         |wx.BORDER_SUNKEN)
		self.list_ctrl.InsertColumn(0, 'First name')
		self.list_ctrl.InsertColumn(1, 'Last name')
		self.list_ctrl.InsertColumn(2, 'Age')
		self.list_ctrl.Append(("john", "", 25))
		self.list_ctrl.Append(("steve", "jobs", 30))
if __name__ == "__main__":
	app = wx.App(False)
	frame = myForm()
	frame.Show()
	app.MainLoop()
```

**Comment by Adriani90** [comment]
cc: @leonardder is this maybe because empty columns are invisible? Could be affected by #9873. However, I do not see why someone needs to have empty columns in lists reported by NVDA.

**Comment by beqabeqa473** [comment]
yes, for some people including me it is important.

because i can jump for example to last name which for example on this
specific item in null but on another items it can be filled.

and also, i don't think they are invisible, they are just empty for
one specific item, but not for other.

On 12/13/19, Adriani90 <notifications@github.com> wrote:
> cc: @leonardder is this maybe because empty columns are invisible? Could be
> affected by #9873. However, I do not see why someone needs to have empty
> columns in lists reported by NVDA.
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/10615#issuecomment-565190524


-- 
with best regards beqa

**Comment by leonardder** [comment]
This indeed looks like a regression. It makes sense to skip the columns when they are of width 0, but in this case, that shouldn't be the case.

**Comment by leonardder** [comment]
Indeed caused by #9873.

**Comment by leonardder** [comment]
Should be fixed in #10623

**Comment by Adriani90** [comment]
Closing as fixed.
